### DISCUSSION TITLE
Api URL to find all github repos for user?

### DISCUSSION THREAD

**Discussion by eopb** [community/community#24393] [Apps, API and Webhooks]
<p><a href="https://api.github.com/users/ethanboxx/repos" rel="nofollow noopener">https://api.github.com/users/ethanboxx/repos</a></p>
<p>Gives 30 results but I have 50 repositories on that github page. Is there a url I could use to get all 50 of them in json form?</p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/ethanboxx">@ethanboxx</a> </p>
<p>Using the below url you can get 50 data set.</p>
<p><a href="https://api.github.com/users/ethanboxx/repos?per_page=50" rel="nofollow noopener">https://api.github.com/users/ethanboxx/repos?per_page=50</a></p>
<p>GitHub has pagination and by default, the result set is 30 only. Either you can traverse through pages or increase the per_page parameter.</p>
<p><a href="https://developer.github.com/v3/#pagination" rel="nofollow noopener">https://developer.github.com/v3/#pagination</a></p>
<p>Regards</p>
<p>Parithiban</p>
